src/test/java/roomescape/service/ReservationServiceTest.java

```diff
@@ -0,0 +1,42 @@
+package roomescape.service;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.annotation.DirtiesContext;
+import roomescape.dto.ReservationRequest;
+import roomescape.dto.ReservationTimeRequest;
+import roomescape.entity.Reservation;
+
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.DEFINED_PORT)
+@DirtiesContext(classMode = DirtiesContext.ClassMode.BEFORE_EACH_TEST_METHOD)
+class ReservationServiceTest {
```

Comment:
`webEnvironment`, `@DirtiesContext`는 없어도 되겠네요.

```suggestion
@SpringBootTest
```